remote-drivers/lwip-ethernet-async/camkes-include/lwip-ethernet-async-control.idl4

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020, Data61
+ * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+ * ABN 41 687 119 230.
+ *
+ * This software may be distributed and modified according to the terms of
+ * the BSD 2-Clause license. Note that NO WARRANTY is provided.
+ * See "LICENSE_BSD2.txt" for details.
+ *
+ * @TAG(DATA61_BSD)
+ */
+
+/* This needs to be here for the mac procedure */
+import <VirtQueue/VirtQueue.camkes>;
+
+procedure lwip_ethernet_async_control {
+    void mac(out uint8_t b1, out uint8_t b2, out uint8_t b3, out uint8_t b4, out uint8_t b5, out uint8_t b6);
```

Comment:
Parameter handling get much simpler if we just use an `uint64_t` for the MAC and define this to use network byte order. Compilers can handle u64 and bit-masking even on 32-bit systems nicely.